pkg/client/unversioned/clientcmd/api/v1/types.go

```diff
@@ -58,6 +58,8 @@ type Preferences struct {
 type Cluster struct {
 	// Server is the address of the kubernetes cluster (https://hostname:port).
 	Server string `json:"server"`
+	// Servers is an array of addresses to the kubernetes apiservers with the format: (https://hostname:port)
```

Comment:
Needs doc about how server overrides this.  It might be better to make this alternateServers so that it's clear which one should be first